vortex-layout/src/layouts/dict/reader.rs

```diff
@@ -328,7 +328,11 @@ mod tests {
         .unwrap()
         .into_array();
         let actual = actual.into_arrow_preferred().unwrap();
-        let expected = expected.into_arrow_preferred().unwrap();
+        // TODO: The expected array has vortex.varbin encoding, which prefers
```

Comment:
This was surprising to me: https://github.com/vortex-data/vortex/blob/11c9452e486d57e01f0560f9e7eed29dcf78fac3/vortex-array/src/arrow/compute/to_arrow/varbin.rs#L90. Shouldn't varbin arrays be zero-copy converted into Utf8Views?

Reply:
You need to do a linear scan to read every string and construct its view into a new views buffer, so it's actually fairly expensive to do this

Reply:
urgh sorry, I was getting confused with the `VarBinView` array. Will remove the TODO